pkg/commands/drift/drift.go

```diff
@@ -329,9 +330,9 @@ func differenceMap(left, right map[string]*assetinventory.AssetIAM) map[string]s
 	return found
 }
 
-// differenceSet finds the keys located in the left set that are missing in the right set.
+// DifferenceSet finds the keys located in the left set that are missing in the right set.
 // We return a set so that we can do future comparisons easily with the result.
-func differenceSet(left, right map[string]struct{}) map[string]struct{} {
+func DifferenceSet(left, right map[string]struct{}) map[string]struct{} {
```

Comment:
We've got [set functions in pkg now](https://github.com/abcxyz/pkg/blob/7ad5bd201d223baaa666fec40b1d13f4f2d571a3/sets/maps.go#L117). Do any of those work?

Reply:
Done